README.md

```diff
@@ -119,7 +121,40 @@ params.SplitCount = 2
 // MinSplitSize default value: 5120
 params.MinSplitSize = 7168
 
-rtManager.DownloadFiles(params)
+totalDownloaded, totalExpected, err := rtManager.DownloadFiles(params)
+```
+
+##### Downloading Files with Results Reader:
+Using `DownloadFilesWithResultReader()` service, we can download files and get the general statistics of the action (The actual number of files downloaded, and the number of files we expected to download), and the error value if it occurred.
+In addition, this service provides us with a reader through which we can iterate the downloaded files information
+```go
+params := services.NewDownloadParams()
+params.Pattern = "repo/*/*.zip"
+params.Target = "target/path/"
+params.Recursive = true
+params.IncludeDirs = false
+params.Flat = false
+params.Explode = false
+params.Symlink = true
+params.ValidateSymlink = false
+// Retries default value: 3
+params.Retries = 5
+// SplitCount default value: 3
+params.SplitCount = 2
+// MinSplitSize default value: 5120
+params.MinSplitSize = 7168
```

Comment:
No need to provide the above params, because we're referencing the user to the previous section.